dcp-0006/dcp-0006.mediawiki

```diff
@@ -338,6 +337,39 @@ The following blockchain parameters were added:
 * Treasury Vote Required Multiplier, or TVRM: 3 (60% yes votes required)
 * Treasury Vote Required Divisor, or TVRD: 5
 
+==Deployment==
+
+===Voting Agenda Parameters===
```

Comment:
Comparing with other DCPs, this header is not usually present.

Reply:
Right.  The next PR (#32) adds it to all the others to differentiate between it and the newly added voting results.

I suppose I can leave it off of this one and add it in along with the others.

Reply:
Done.